fastlane_core/spec/analytics/analytics_session_spec.rb

```diff
@@ -38,7 +43,11 @@
         expect(session).to receive(:ide_version).and_return('Xcode 9')
 
         session.action_launched(launch_context: launch_context)
-        expect(JSON.parse(session.events.to_json)).to match_array(fixture_data)
+
+        parsed_events = JSON.parse(session.events.to_json)
+        parsed_events.zip(fixture_data).each do |parsed, fixture|
+          expect(parsed).to eq(fixture)
```

Comment:
This is cool, just curious why using this over the rspec `match_array` matcher?

Reply:
when `match_array` fails, the output is a giant string describing the differences. It's pretty close to impossible to identify what is difference. By iterating through each object you can zero in on the difference immediately.